src/renderer/css/modals-dark.scss

```diff
@@ -4,6 +4,18 @@
     @include meta.load-css('bootstrap-dark/src/bootstrap-dark.css');
```

Comment:
we still are using the bootstrap css?

Reply:
It still needs to be there for the Settings Modal, we've removed reference to this CSS for all the other modals.